### PR TITLE
Rename schema file to data.schema.json and publish it

### PR DESCRIPTION
**Pull request by foolip**
This will include the schema in the NPM package. The idea is to also 
publish it with GitHub releases.

### CODE REVIEW

**Comment by foolip** [comment]
The idea is that this would help with https://github.com/web-platform-dx/web-features/issues/715. We can publish web-features.schema.json in both the NPM package and the GitHub release. The name of the data files are different (index.json vs. data.json) so matching either of those names would still leave a mismatch in the other place.

**Review by ddbeck — APPROVED** [review]
Ah, so the end-goal here is to export this on the package? `import { schema } from "web-features"`? In any case, this is fine.

**Comment by foolip** [comment]
I've sent https://github.com/web-platform-dx/web-features/pull/1470, and if we do that I think we should call this `data.schema.json` instead.

File: package.json [hunk 1]
```diff
@@ -20,8 +20,8 @@
     "dist": "tsx scripts/dist.ts",
     "feature-init": "tsx scripts/feature-init.ts",
     "format": "npx prettier --write .",
-    "schema-defs:write": "npm run schema-defs -- --out ./schemas/defs.schema.json",
-    "schema-defs": "ts-json-schema-generator --tsconfig ./tsconfig.json --type WebFeaturesData --path ./types.ts --id defs",
+    "schema:write": "npm run schema -- --out ./schemas/data.schema.json",
+    "schema": "ts-json-schema-generator --tsconfig ./tsconfig.json --type WebFeaturesData --path ./types.ts",
```
Comment (ddbeck) [inline package.json:24]: This is no longer needed either (it has its own name one level down in definitions).

```suggestion
    "schema": "ts-json-schema-generator --tsconfig ./tsconfig.json --path ./types.ts",
```

**Review by ddbeck — APPROVED** [review]
It'd be nice to update `packages/web-features/README.md` with an example `import` statement for this, but I won't block on it. One minor suggestion in a line comment.

**Comment by foolip** [comment]
I've documented the import as `import schema from "web-features/data.schema.json" with { type: "json" }` and will merge this now.

**Pull request merged**